stdlib/private/SwiftPrivate/SwiftPrivate.swift

```diff
@@ -13,69 +13,69 @@
 import SwiftShims
 
 /// Convert the given numeric value to a hexadecimal string.
-  // FIXME(integers): support a more general BinaryInteger protocol
+// FIXME(integers): support a more general BinaryInteger protocol
 public func asHex<T : FixedWidthInteger>(_ x: T) -> String {
-  return "0x" + String(x, radix: 16)
+    return "0x\(String(x, radix: 16))"
 }
 
 /// Convert the given sequence of numeric values to a string representing
 /// their hexadecimal values.
-  // FIXME(integers): support a more general BinaryInteger protocol
+// FIXME(integers): support a more general BinaryInteger protocol
 public func asHex<S : Sequence>(_ x: S) -> String
-  where
-  S.Element : FixedWidthInteger {
-  return "[ " + x.lazy.map { asHex($0) }.joined(separator: ", ") + " ]"
```

Comment:
Can you change these indentations back to 2 spaces? The Swift standard library prefers 2-space indentation.